pages.es/common/grep.md

```diff
@@ -0,0 +1,37 @@
+# grep
+
+> Encuentra coincidencias en el texto introducido.
+> Soporta patrones simples y expresiones regulares.
+> Más información: <https://www.gnu.org/software/coreutils/grep/manual>
```

Comment:
```suggestion
> Más información: <https://www.gnu.org/software/grep/manual/>
```

Reply:
Actually I think <https://www.gnu.org/software/grep/manual/grep.html> would be even better than <https://www.gnu.org/software/grep/manual/>.